### DISCUSSION TITLE
0.2.9

### DISCUSSION THREAD

**Discussion by schickling** [contentlayerdev/contentlayer#328] [Announcements]
### Next.js 13 Support

Slightly delayed (sorry about that) Contentlayer now finally supports the Next.js version 13. Things should work just as they did before when using `getStaticProps`. 🚀

However, unfortunately React Server Components (RSC) can't yet be used with Contentlayer as there's a number of blocking bugs in Next.js itself (e.g. https://github.com/vercel/next.js/issues/41865) which need to be fixed first. You can track the progress here: https://github.com/contentlayerdev/contentlayer/issues/311

### Other changes

- Improved types for `useMDXComponent` (https://github.com/contentlayerdev/contentlayer/pull/312 - thanks @Andrey-Bazhanov)
- Upgraded other deps

<hr /><em>This discussion was created from the release <a href='https://github.com/contentlayerdev/contentlayer/releases/tag/v0.2.9'>0.2.9</a>.</em>

**Comment by gipsterya** [comment]
Thank you @schickling. Now I’m going to ship it 🚀🚀🚀

**Comment by mflisikowski** [comment]
Hi, I encountered such an error today after updating

<img width="1237" alt="image" src="https://user-images.githubusercontent.com/1973631/200630645-7ae236fc-c6a7-4c02-b618-bd51e2af2fad.png">

Screen after update:

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/1973631/200631630-6e401b47-83fd-4f3b-b679-b8b17e258eeb.png">

Screen after downgrade:

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/1973631/200631957-14a573a8-155a-4754-920f-aa52d1e1e1bf.png">

**Reply by mflisikowski** [reply]
https://github.com/contentlayerdev/contentlayer/issues/329
